FreeRTOS/FreeRTOS
# Update history for release [202411.00]

Update history for release [202411.00]

Description
-----------
This PR updates history for release [202411.00] and updates the submodule pointer of corePKCS11 to point to v3.6.2.

Test Steps
-----------
Ran the [corePKCS11_MQTT_Mutual_Auth_Windows_Simulator](https://github.com/FreeRTOS/FreeRTOS/tree/p3_rel_wip/FreeRTOS-Plus/Demo/corePKCS11_MQTT_Mutual_Auth_Windows_Simulator)

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
